### PR TITLE
Add note about tombstones appearing in the top-level of the BSO

### PR DESCRIPTION
**Pull request by mhammond**

### CODE REVIEW

**Review by rfk — APPROVED** [review]
thanks!

**Comment by rfk** [comment]
Seems safe to merge this even while the patches are under review, rather than wait until they've landed and then maybe forget about this...@mhammond WDYT?

**Comment by mhammond** [comment]
Too late, but yeah - that's why I opened the app-services issue when I did, so I could link to it before it's complete.

**Pull request merged**
